src/portal_to_terra_input_from_anaset.py

```diff
@@ -0,0 +1,298 @@
+#     """
+#     if igvf_site == 'production':
+#         host = 'https://api.data.igvf.org'
+#     elif igvf_site == 'sandbox':
+#         host = 'https://api.sandbox.igvf.org'
+#     config = Configuration(
+#         access_key=os.environ["IGVF_API_KEY"],
+#         secret_access_key=os.environ["IGVF_SECRET_KEY"],
+#         host=host
+#     )
+#     client = ApiClient(config)
+#     return IgvfApi(client)
+
+
+#  HACK: A map that translates an assay term to either ATAC or RNA
+# assay_terms_map = {
+#     '/assay-terms/OBI_0003109/': 'rna',
+#     '/assay-terms/OBI_0002762/': 'atac',
+#     '/assay-terms/OBI_0002764/': 'atac',
+#     '/assay-terms/OBI_0002631/': 'rna',
+#     '/assay-terms/OBI_0003090/': 'rna',     # For test analysis set only
+#     '/assay-terms/OBI_0003089/': 'atac'     # For test analysis set only
+# }
+
+# HACK: A map that uses platform_preferred-assay-title_assay-type to inference read map for pipelines.
+read_type_by_platform_map = {
+    '/platform-terms/EFO_0008637/_SHARE-seq_atac': {'R1': 'read1', 'R2': 'read2'},
+    '/platform-terms/EFO_0008637/_SHARE-seq_rna': {'R1': 'read1', 'R2': "read2"},
+    '/platform-terms/EFO_0022840/_10x multiome with MULTI-seq_atac': {'R1': 'read1', 'R3': 'read2'},
+    '/platform-terms/EFO_0022840/_10x multiome with MULTI-seq_rna': {'R1': 'read1', 'R2': 'read2'},
+    '/platform-terms/EFO_0008637/_RNA-seq_rna': {'R1': 'read1', 'R2': 'read2'},     # For test analysis set only
+    '/platform-terms/EFO_0010963/_ATAC-seq_atac': {'R1': 'read1', 'R2': 'read2'},    # For test analysis set only
+    '/platform-terms/NTR_0000765/_10x multiome with MULTI-seq_rna': {'R1': 'read1', 'R2': 'read2'},
+    '/platform-terms/NTR_0000765/_10x multiome with MULTI-seq_atac': {'R1': 'read1', 'R3': 'read2'},
+    '/platform-terms/EFO_0010963/_Parse SPLiT-seq_rna': {'R1': 'read1', 'R2': 'read2'}
+}
+
+
+def get_assay_type_from_assay_id(assay_term_id: str, igvf_api) -> str:
+    # NOTE: This may get more complicated if the pipeline wants to distinguish sn vs. sc
+    """Infer assay type (RNA or ATAC) from the assay term summary
+
+    Args:
+        assay_term_id (str): assay term id
+        igvf_api (_type_): _description_
+
+    Returns:
+        str: rna or atac
+    """
+    assay_term_summary = igvf_api.get_by_id(assay_term_id).actual_instance.summary
+    if 'RNA' in assay_term_summary:
+        return 'rna'
+    elif 'ATAC' in assay_term_summary:
+        return 'atac'
+
+
+def get_samp_taxa_from_samp_id(sample_id: str, igvf_api) -> str:
+    """Get sample taxa from a sample ID
+
+    Args:
+        sample_id (str): e.g., /in-vitro-system/IGVFxxxxx
+        igvf_api (_type_): _description_
+
+    Returns:
+        str: e.g., Homo sapiens
+    """
+    sample_item = igvf_api.get_by_id(sample_id).actual_instance
+    return sample_item.taxa
+
+
+def get_measet_ids_and_other_info_from_analysis_set_acc(analysis_set_accessions: list, igvf_api) -> tuple:
```

Comment:
Can separate this function into two functions? one return measurement_ids, one return sample_taxa.